scanner/src/lib/dataproducts/Battery_CarbonFootprint_v0_1.svelte

```diff
@@ -75,28 +78,5 @@
   article {
```

Comment:
It's the repetitive part at least around 2 files. Probably should be moved to separate component